olp-cpp-sdk-core/include/olp/core/network2/NetworkTypes.h

```diff
@@ -21,28 +21,29 @@
 
 #include <cstdint>
 #include <limits>
+#include <type_traits>
 
 namespace olp {
 namespace network2 {
 
-/**
- * @brief Represents request id.
- * Values of this type are returned by Network::Send and used by
- * Network::Cancel.
- */
-using RequestId = std::uint64_t;
-
 /**
  * @brief List of special values for NetworkRequestId.
  */
-enum class RequestIdConstants : std::uint64_t {
```

Comment:
please leave enum class, see [here](https://github.com/heremaps/here-olp-edge-sdk-cpp/blob/master/CONTRIBUTING.md)

Reply:
enum class is useful for "true" enumerations - it makes harder to use non-existing enum values.
On other hand, RequestID in meant to use with an increment, see its usage in NetworkCurlImpl